NitroxClient/GameLogic/RemotePlayer.cs

```diff
@@ -17,6 +17,10 @@ namespace NitroxClient.GameLogic
 {
     public class RemotePlayer : INitroxPlayer
     {
+        /// <summary>
+        /// Marks <see cref="Player.mainObject"/> and every <see cref="Body"/> so they can be precisely queried (e.g. by sea dragons).
+        /// </summary>
+        public const EcoTargetType PLAYER_ECO_TARGET_TYPE = (EcoTargetType)5050;
```

Comment:
Looking at the EcoTargetType enum, not sure to fully understand where the 5050 is coming from, maybe we can add some details in the comments